package.json

```
@@ -105,7 +105,7 @@
         "vite-plugin-eslint": "1.8.1"
     },
     "dependencies": {
-        "@frontify/fondue-tokens": "3.1.0",
+        "@frontify/fondue-tokens": "^3.2.0",
```

```suggestion
        "@frontify/fondue-tokens": "3.2.0",
```
Please pin the dependency to avoid any issue :)

---

Thanks a lot! :D